python/tvm/relay/base.py

```diff
@@ -51,9 +51,12 @@ def astext(self, show_meta_data=True, annotate=None):
             Optional annotate function to provide additional
             information in the comment block.
 
+        gnf : bool
+            Whether to print in GNF.
+
         Note
         ----
-        The metadata section is necessary to fully parse the text format.
```

Comment:
https://en.wikipedia.org/wiki/Metadata
metadata is a word

Reply:
I agree, but meta data is currently far more common in the code base.